jest/fullscreen.spec.js

```diff
@@ -0,0 +1,52 @@
+require("../app/assets/javascripts/fullscreen.js")
+
+let _window = {innerWidth: 600, innerHeight:400}
+let _screen = {width: 1200, height: 800 }
+let transform = {}
+const maxWidth = 2000; // Hidden constant in fullscreen.js
+
+describe("GetIframeTransforms", () => {
+
+  it("should exist", () => {
+    expect(global.GetIframeTransforms).toBeDefined()
+  })
+
+  describe("Normal sized screens", () => {
+    beforeEach(()=> {
+      transform = global.GetIframeTransforms(_window, _screen)
+    });
+    it('transforms', ()=> {
+      expect(transform.width).toBe(1200)
+      expect(transform.scale).toBe(_window.innerWidth / _screen.width)
+      expect(transform.height).toBe(_window.innerHeight / transform.scale)
```

Comment:
I'm not totally sure, but I think it would be clearer if you used actual numbers here.  You can use jest's `toBeCloseTo` to do approximate floating point comparisons: https://jestjs.io/docs/en/expect#tobeclosetonumber-numdigits

By default it uses 2 decimal places, which is probably good enough for our test cases.
So in this case it would be:
```
expect(transform.scale).toBeCloseTo(0.5)
expect(transform.height).toBeCloseTo(800)
```

Reply:
I added the numeric assertions as you suggested.  This must be a case of personal style, because i find the explicit calculation easier to reason about, so  I left those assertions there too.